src/components/shared/CallNodeContextMenu.css

```diff
@@ -26,26 +26,14 @@
 .callNodeContextMenuLabel {
   color: #555;
   font-weight: bold;
-  margin-inline: 2px;
```

Comment:
This was the "magic" that made the improper spacing around the library name `libxul.so`. This was needed because it was just a flex item and as a result was positioned without any space. Instead I've put all the text in another wrapping div that doesn't use flex layout.